pkg/proxy/ipvs/proxier.go

```diff
@@ -1090,15 +850,25 @@ func (proxier *Proxier) syncProxyRules() {
 	}
 
 	// Build IPVS rules for each service.
-	for svcName, svcInfo := range proxier.serviceMap {
+	for svcName, svc := range proxier.serviceMap {
+		svcInfo, ok := svc.(*serviceInfo)
+		if !ok {
+			glog.Errorf("Failed to cast serviceInfo")
```

Comment:
Would be useful to include svcName in the log message

Reply:
Done.